std_msgs/Time stamp
int64 a
---
float32[] desc
string model_type
